core/src/bms/player/beatoraja/launcher/PlayConfigurationView.fxml

```diff
@@ -415,6 +415,33 @@
                         </children>
                     </GridPane>
 
+                    <VBox prefHeight="25.0" prefWidth="300.0">
+                        <VBox.margin>
+                            <Insets top="10.0" />
+                        </VBox.margin>
+                        <CheckBox fx:id="discord" mnemonicParsing="false" prefHeight="25.0" text="%DISCORD_RPC" />
+                        <CheckBox fx:id="clipboardScreenshot" mnemonicParsing="false" prefHeight="25.0" text="%CLIPBOARD_SCREENSHOT" />
+                        <CheckBox fx:id="enableWebhook" mnemonicParsing="false" prefHeight="25.0" text="Enable Discord Webhook on screenshot" />
+                        <GridPane hgap="10" vgap="5">
+                            <children>
+                                <Label text="Discord Webhook Name" prefHeight="25.0" GridPane.rowIndex="0" GridPane.columnIndex="0" />
+                                <TextField fx:id="webhookName" prefHeight="25.0" prefWidth="400" GridPane.rowIndex="0" GridPane.columnIndex="1" />
+                            </children>
+                        </GridPane>
+                        <GridPane hgap="10" vgap="5">
+                            <children>
+                                <Label text="Discord Webhook Avatar" prefHeight="25.0" GridPane.rowIndex="0" GridPane.columnIndex="0" />
+                                <TextField fx:id="webhookAvatar" prefHeight="25.0" prefWidth="400" GridPane.rowIndex="0" GridPane.columnIndex="1" />
+                            </children>
+                        </GridPane>
+                        <GridPane hgap="10" vgap="5">
+                            <children>
+                                <Label text="Discord Webhook URL" prefHeight="25.0" GridPane.rowIndex="0" GridPane.columnIndex="0" />
+                                <PasswordField fx:id="webhookUrl" prefHeight="25.0" prefWidth="400" GridPane.rowIndex="0" GridPane.columnIndex="1" />
```

Comment:
Pointers on turning this into an array field would be greatly appreciated! I looked at how the tables tab does it and it seems overly complex? Is that just how it is?